app/precompiled.py

```diff
@@ -105,6 +105,39 @@ def rect(self, pt1, pt2):
         super().rect(left_x, bottom_y_from_bottom, width, height, fill=True, stroke=False)
 
 
+class PostalAddress(PostalAddress):
```

Comment:
Optional: I'm not sure if I like the inheriting a class with the same name. I feel it make things a bit opaque but I'm not decided

Reply:
I’ve renamed it to `PrecompiledPostalAddress` (since only precompiled letters need the address as a regex, for example).